### PR TITLE
Cm 60 add nightly reconcilliation

### PR DESCRIPTION
**Pull request by dcmcand**
**Description of change**
Adds job to worker to process legacy report reconciliation

**How to test**
uncomment line 165 of `src/services/legacyreports.js` and comment out line 166. That will make the reconciliation run every minute instead of every night at 2am. Start the app and watch the worker logs. You should see it start the reconciliation process every minute.

If you import the legacy reports and change the users email addresses as describted in #255, you should see the legacy reports get updated within a minute of changing the email address.

**Issue(s)**
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-72
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-60

**Checklist**

- [X] Meets issue criteria
- [X] Code tested
- [ ] Meets accessibility standards (WCAG 2.1 Levels A, AA)
- [ ] Documentation updated
    - API methods
    - Boundary and Data Flow Diagrams
    - [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions with the [Nygard template](https://github.com/joelparkerhenderson/architecture_decision_record/blob/master/adr_template_by_michael_nygard.md)
    - OSCAL templates completed when security controls are implemented or modified

### CODE REVIEW

**Review by dcloud — APPROVED** [review]
This worked like a charm when testing with the every minute cron and the changes per the previous ticket. Nice work!

File: src/worker.js [hunk 1]
```diff
@@ -1,9 +1,10 @@
 /* eslint-disable import/first */
-require('newrelic');
+// require('newrelic');
```
Comment (rahearn) [inline src/worker.js:2]: I think this should probably not be commented out.

File: src/worker.js [hunk 2]
```diff
@@ -23,6 +24,19 @@ function start() {
...
+
+  reconciliationQueue.on('failed', (job, error) => auditLogger
+    .error(`${job.data.key}: Legacy report reconciliation failed with error ${error}`));
+  reconciliationQueue.on('completed', () => logger
+    .info('Legacy report reconciliation completed successfully'));
+  reconciliationQueue.process('legacyReports', async (job) => {
+    logger.info(`starting ${job}`);
+    try {
+      await reconcileLegacyReports();
+    } catch (e) {
```
Comment (rahearn) [inline src/worker.js:36]: Does catching this error prevent the `on('failed'` callback from being run? Is there a reason that'd be better?

**Review by rahearn — CHANGES_REQUESTED** [review]

File: src/worker.js (hunk 2 above)
Comment (dcmcand) [reply src/worker.js:36]: it was just so I could see the error message in the console when debugging. It shouldn't catch it in prod. nice catch there.

File: src/worker.js (hunk 1 above)
Comment (dcmcand) [reply src/worker.js:2]: you are, of course, correct

**Review by rahearn — APPROVED** [review]

**Pull request merged**